### PR TITLE
[[Bug 11796]] iPhoneFileDataProtection empty result

### PR DESCRIPTION
**Pull request by seaniepie**
- Added tFile to the function in the example
- Added description to all results
- Added empty to the list of results

### CODE REVIEW

File: docs/dictionary/function/iphoneFileDataProtection.lcdoc [hunk 1]
```diff
@@ -24,10 +24,11 @@ A string containing the path to the file.
...
 
--   none: 
--   complete: 
--   complete unless open: (iOS 5 and later only)
--   complete until first user authentication: (iOS 5 and later only)
+-   empty: The file was not accessible or argument not specified.
+-   none: No protection.
+-   complete: The file is not accessible, for read or write, while the device is locked.
+-   complete unless open: The file is fully protected when the device is locked, unless it was already open.
+-   complete until first user authentication: The file is fully protected until the user unlocks the device for this first time.
```
Comment (livecodeali) [inline docs/dictionary/function/iphoneFileDataProtection.lcdoc:32]: The values here (other than `empty`) need to be in double-quotes.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 7cdefa0

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 7cdefa0610eb73e6d62d0e9fbcde7e3e44aae8f9

**Comment by livecode-vulcan** [comment]
😎 test success 7cdefa0610eb73e6d62d0e9fbcde7e3e44aae8f9
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/476)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/481)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/476)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/476)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/482)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/476)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/476)

**Pull request merged**